features/sensitive.ui.feature

```diff
@@ -0,0 +1,82 @@
+# @licence GNU GPL v2+
+# @author Christoph Fischer <christoph.fischer@wikimedia.de>
+
+Feature: Checks wikimedia.de fundraising ui functionality in the sensitive banner for Wikipedia
+
+  Background:
+    When I am on a random Wikipedia article page and provide a B15WMDE_sensitive
+
+  Scenario: Checks if the banner shows
+    Then The WMDE_BannerFullForm should become visible
+
+  Scenario: Checks if the form switches the interval options
+    When WMDE_BannerFullForm becomes visible
+    And I click the regularly interval option
+    Then Regularly details shows
+
+  Scenario: Checks if the deposit payment method opens correctly
+    When WMDE_BannerFullForm becomes visible
+    And I click sensitive banner debit option
+    Then The sepa donation part should be visible
+    And The nonsepa donation part should not be visible
+    And The person donation part should be visible
+    And The company donation part should not be visible
+    And The address donation part should be visible
+    And The next button should be visible
+    And The finish donation button should not be visible
+
+  Scenario Outline: Checks if the non-deposit payment methods open correctly
```

Comment:
Shouldn't it say `non-**debit** payment methods` here? One of payment_methods listed in examples is indeed "deposit". And if this comment is right, line 17 above should then say `Checks if the debit payment`, I guess.